tensorflow/python/keras/backend.py

```diff
@@ -2918,6 +3001,19 @@ def stack(x, axis=0):
 
   Returns:
       A tensor.
+
+  Example:
+      ```python
+        >>> a = tf.constant([[1, 2],[3, 4]])
+        >>> b = tf.constant([[10, 20],[30, 40]])
+        >>> tf.keras.backend.stack((a, b))
+        <tf.Tensor: id=146, shape=(2, 2, 2), dtype=int32, numpy=
+        array([[[ 1,  2],
+                [ 3,  4]],
+        
```

Comment:
what is the blank line here? is it supposed to be "..."?

Reply:
The empty lines are returned by Python. I deleted them in 7d2750d to remove unnecessary lines.